pom.xml

```diff
@@ -220,6 +219,15 @@
         <error_prone_core.version>1.1.2</error_prone_core.version>
         <plexus-compiler-javac.version>2.3</plexus-compiler-javac.version>
         <plexus-compiler-javac-errorprone.version>2.3</plexus-compiler-javac-errorprone.version>
+
+        <!-- Cloud Express stuff -->
+        <rice.version>2.5.2.0-kualico-SNAPSHOT</rice.version>
+        <!-- TODO bind to a release version -->
+        <coeus-api-all.version>6.0.0.1</coeus-api-all.version>
```

Comment:
This property is listed twice.  On the other hand I'm not seeing a property for the s2sgen dependency.